packages/components/src/navigation/stories/index.js

```diff
@@ -48,6 +48,11 @@ function Example() {
 							navigateToMenu="category"
 							title="Category"
 						/>
+						<NavigationItem
+							item="item-pointing-non-existing-menu"
+							title="Navigate to a non existing menu"
```

Comment:
Minor: I've noticed that long labels are displayed centered.
Would you mind adding a `text-align: left` to `ItemTitleUI` in the [navigation styles](https://github.com/WordPress/gutenberg/blob/7c467c773b7c391bfe40281a39ac5630688e3437/packages/components/src/navigation/styles/navigation-styles.js#L100-L102)?
It's unrelated to this change, but it's a one-liner that would prevent that awkwardness in the story example.

Reply:
Yep! Noticed that one too but for some reason, I thought it's intentional or something like that :D I wanted to bring it up but totally forgot about it... Thanks!